types/stylelint/index.d.ts

```diff
@@ -251,6 +252,22 @@ declare namespace stylelint {
 		fixable?: boolean;
 	};
 
+	export type Range = {
+		start: {
+			line: number;
+			column: number /** inclusive */;
+		};
+		end: {
+			line: number;
+			column: number /** exclusive */;
+		};
+	};
```

Comment:
[suggest] We already have the same data structure for line and column here:

https://github.com/stylelint/stylelint/blob/06e93eecfb0bea0823a1544d873232a12b7d7fa6/types/stylelint/index.d.ts#L617-L629

So, it seems to make sense to add a new data type like PostCSS's [`Position`](https://postcss.org/api/#position). What do you think?

```ts
type Position = {
  line: number;
  column: number;
};
```

Reply:
I didn't add a common type because I wanted to document the inclusive/exclusive difference inline.
I guess we could add `Position`, yes.

Reply:
So, let's add `Position`. If it's added, the `Problem` and `Range` types should be as follows:

```ts
type Problem = {
  // ...
  start?: Position;
  end?: Position;
  // ...
};

type Range = {
  start: Position;
  end: Position;
};
```